torch/csrc/Generator.cpp

```diff
@@ -68,7 +69,7 @@ static PyObject * THPGenerator_getState(THPGenerator *self)
   using namespace torch::autograd;
   HANDLE_TH_ERRORS
   THGenerator *generator = THPGenerator_TH_CData(self);
-  Variable var = VariableType::getType(CPU(kByte))->tensor();
+  Variable var = torch::empty({0}, at::device(at::kCPU).dtype(at::kByte));
```

Comment:
can remove `at::device(at::kCPU)` (it's default), then just pass `at::kByte` for the whole thing

Reply:
I don't think removing `at::kCPU` here is wise: someone may have set the default to CUDA, but we definitely positively want a CPU tensor here...